scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -160,15 +128,56 @@ private void changeZValue(){
     double zValue = zValueSlider.getValue();
 
     // update the geometry of each of the existing graphics to include the new z-value
-    for (GraphicsOverlay graphicsOverlay : sceneView.getGraphicsOverlays()) {
-      for (Graphic graphic : graphicsOverlay.getGraphics()) {
+    sceneView.getGraphicsOverlays().forEach(graphicsOverlay ->
+      graphicsOverlay.getGraphics().stream().filter(
+        graphic -> graphic.getGeometry().getGeometryType() == GeometryType.POINT).forEach(graphic -> {
         Geometry geometry = graphic.getGeometry();
-        if (geometry.getGeometryType().equals(GeometryType.POINT)) {
-          Point currentPoint = (Point) geometry;
-          Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
-          graphic.setGeometry(updatedPoint);
-        }
-      }
+        Point currentPoint = (Point) geometry;
+        Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
+        graphic.setGeometry(updatedPoint);
+      }));
```

Comment:
sorry this was my fault: we should do some breaking up of these brackets here to be more java-y :) 

```java
  /**
   * Sets the Z-Value to the value selected by the slider.
   */
  @FXML
  private void changeZValue() {
    // get the z-value from the slider
    double zValue = zValueSlider.getValue();

    // update the geometry of each of the existing graphics to include the new z-value
    sceneView.getGraphicsOverlays().forEach(graphicsOverlay ->
      graphicsOverlay.getGraphics().stream().filter(
        graphic -> graphic.getGeometry().getGeometryType() == GeometryType.POINT).forEach(graphic -> {
        Geometry geometry = graphic.getGeometry();
        Point currentPoint = (Point) geometry;
        Point updatedPoint = new Point(currentPoint.getX(), currentPoint.getY(), zValue, currentPoint.getSpatialReference());
        graphic.setGeometry(updatedPoint);
      })
    );
  }

```